src/common/Parameter.cpp

```diff
@@ -3104,12 +3118,16 @@ void Parameter::get_display(char *txt, bool external, float ef) const
         case DelegatedToFormatter:
         {
             auto ef = dynamic_cast<ParameterExternalFormatter *>(user_data);
+            char str[TXT_SIZE];
+
             if (ef)
             {
                 // parameter called 'len' not 'size', be on the safe side here, do - 1.
                 // It used to say just '64' anyway.
-                if (ef->formatValue(this, f, txt, TXT_SIZE - 1))
-                    return;
+                if (ef->formatValue(this, f, str, TXT_SIZE - 1))
+                {
+                    return txt;
```

Comment:
it did a void return but it had mutated txt.

so the way this works is formatValue sets onto text. text is passed in. so text is modified at the return

the way your change makes it work os formatValue modifies str but then ignores that modification and returns txt 

so the formatting is ignored

I think the equivalent change is `return str` rather than `return txt`